### ISSUE TITLE
Crashes on opening - The GPU process has crashed

### ISSUE THREAD

**Issue by hizeph** [microsoft/vscode#148690]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.66.2
- OS Version: 20.04.1-Ubuntu (5.13.0-40)

Steps to Reproduce:

1. Open code

---

`--no-sandbox` and `--no-gpu-sandbox` make no difference.

Here's the output from `code --crash-reporter-directory /tmp/code-crash --verbose --log debug --disable-extensions`
```
Found --crash-reporter-directory argument. Setting crashDumps directory to be '/tmp/code-crash'
[32555:0504/093906.248418:WARNING:bluez_dbus_manager.cc(248)] Floss manager not present, cannot set Floss enable/disable.
[main 2022-05-04T08:39:06.308Z] [File Watcher (node.js)] Request to start watching: /home/cezar/.config/Code/User (excludes: <none>),/home/cezar/.config/Code/User/settings.json (excludes: <none>)
[main 2022-05-04T08:39:06.321Z] Starting VS Code
[main 2022-05-04T08:39:06.322Z] from: /usr/share/code/resources/app
[main 2022-05-04T08:39:06.322Z] args: {
  _: [],
  diff: false,
  add: false,
  goto: false,
  'new-window': false,
  'reuse-window': false,
  wait: false,
  help: false,
  'list-extensions': false,
  'show-versions': false,
  'pre-release': false,
  version: false,
  verbose: true,
  log: 'debug',
  status: false,
  'prof-startup': false,
  'no-cached-data': false,
  'prof-v8-extensions': false,
  'disable-extensions': true,
  'disable-gpu': false,
  'ms-enable-electron-run-as-node': false,
  telemetry: false,
  debugRenderer: false,
  logExtensionHostCommunication: false,
  'skip-release-notes': false,
  'skip-welcome': false,
  'disable-telemetry': false,
  'disable-updates': false,
  'disable-keytar': false,
  'disable-workspace-trust': false,
  'disable-crash-reporter': false,
  'crash-reporter-directory': '/tmp/code-crash',
  'skip-add-to-recently-opened': false,
  'unity-launch': false,
  'open-url': false,
  'file-write': false,
  'file-chmod': false,
  'driver-verbose': false,
  force: false,
  'do-not-sync': false,
  trace: false,
  'force-user-env': false,
  'force-disable-user-env': false,
  'open-devtools': false,
  __sandbox: false,
  'no-proxy-server': false,
  'no-sandbox': false,
  nolazy: false,
  'force-renderer-accessibility': false,
  'ignore-certificate-errors': false,
  'allow-insecure-localhost': false,
  'disable-dev-shm-usage': false,
  logsPath: '/home/cezar/.config/Code/logs/20220504T093906'
}
[main 2022-05-04T08:39:06.324Z] Resolving machine identifier...
[main 2022-05-04T08:39:06.324Z] Resolved machine identifier: e565754f13590d1924fefaa01f9a568cd67308e99b25d446c75767a73affdf6e
[main 2022-05-04T08:39:06.325Z] Main->SharedProcess#connect
[main 2022-05-04T08:39:06.329Z] [File Watcher (node.js)] Started watching: '/home/cezar/.config/Code/User'
[main 2022-05-04T08:39:06.331Z] [File Watcher (node.js)] Started watching: '/home/cezar/.config/Code/User/settings.json'
[main 2022-05-04T08:39:06.337Z] StorageMainService: creating global storage
[main 2022-05-04T08:39:06.342Z] lifecycle (main): phase changed (value: 2)
[main 2022-05-04T08:39:06.343Z] windowsManager#open
[main 2022-05-04T08:39:06.344Z] windowsManager#open pathsToOpen [
  {
    workspace: { id: '71ff4fd78a9116c7bb1cb745de05611f', configPath: [h] },
    type: 1,
    exists: true,
    remoteAuthority: undefined,
    transient: undefined
  }
]
[main 2022-05-04T08:39:06.345Z] windowsManager#doOpenFolderOrWorkspace {
  folderOrWorkspace: {
    workspace: { id: '71ff4fd78a9116c7bb1cb745de05611f', configPath: [h] },
    type: 1,
    exists: true,
    remoteAuthority: undefined,
    transient: undefined
  },
  filesToOpen: undefined
}
[main 2022-05-04T08:39:06.346Z] IPC Object URL: Registered new channel vscode:2728797c-c4a9-40bf-ad57-1b9d003059d6.
[main 2022-05-04T08:39:06.346Z] window#validateWindowState: validating window state on 3 display(s) { mode: 0, x: 2120, y: 44, width: 1440, height: 781 }
[main 2022-05-04T08:39:06.346Z] window#validateWindowState: multi-monitor working area { x: 1920, y: 0, width: 1920, height: 1080 }
[main 2022-05-04T08:39:06.346Z] window#ctor: using window state { mode: 0, x: 2120, y: 44, width: 1440, height: 781 }
[main 2022-05-04T08:39:06.394Z] window#load: attempt to load window (id: 1)
[main 2022-05-04T08:39:06.402Z] StorageMainService: creating workspace storage (71ff4fd78a9116c7bb1cb745de05611f)
[main 2022-05-04T08:39:06.402Z] windowsManager#open used window count 1 (workspacesToOpen: 1, foldersToOpen: 0, emptyToRestore: 0, emptyToOpen: 0)
[main 2022-05-04T08:39:06.402Z] lifecycle (main): phase changed (value: 3)
[main 2022-05-04T08:39:06.403Z] update#setState idle
[main 2022-05-04T08:39:06.403Z] resolveShellEnv(): skipped (VSCODE_CLI is set)
[main 2022-05-04T08:39:06.428Z] ExtensionHostStarterWorker created
[main 2022-05-04T08:39:06.508Z] resolveShellEnv(): skipped (VSCODE_CLI is set)
[32555:0504/093906.874384:ERROR:gpu_process_host.cc(983)] GPU process exited unexpectedly: exit_code=133
[32555:0504/093906.874403:WARNING:gpu_process_host.cc(1288)] The GPU process has crashed 1 time(s)
[32555:0504/093907.435032:ERROR:gpu_process_host.cc(983)] GPU process exited unexpectedly: exit_code=133
[32555:0504/093907.435046:WARNING:gpu_process_host.cc(1288)] The GPU process has crashed 2 time(s)
[32555:0504/093908.081655:ERROR:gpu_process_host.cc(983)] GPU process exited unexpectedly: exit_code=133
[32555:0504/093908.081668:WARNING:gpu_process_host.cc(1288)] The GPU process has crashed 3 time(s)
[32555:0504/093908.238576:ERROR:gpu_process_host.cc(983)] GPU process exited unexpectedly: exit_code=133
[32555:0504/093908.238593:WARNING:gpu_process_host.cc(1288)] The GPU process has crashed 4 time(s)
[32555:0504/093908.387269:ERROR:gpu_process_host.cc(983)] GPU process exited unexpectedly: exit_code=133
[32555:0504/093908.387281:WARNING:gpu_process_host.cc(1288)] The GPU process has crashed 5 time(s)
[32555:0504/093908.536032:ERROR:gpu_process_host.cc(983)] GPU process exited unexpectedly: exit_code=133
[32555:0504/093908.536044:WARNING:gpu_process_host.cc(1288)] The GPU process has crashed 6 time(s)
[32683:0504/093908.541624:WARNING:vaapi_wrapper.cc(605)] VAAPI video acceleration not available for disabled
[32683:0504/093908.541727:ERROR:gpu_init.cc(454)] Passthrough is not supported, GL is disabled, ANGLE is
[32555:0504/093908.553562:WARNING:gpu_process_host.cc(1011)] Reinitialized the GPU process after a crash. The reported initialization time was 0 ms
[main 2022-05-04T08:39:36.404Z] update#checkForUpdates, state =  idle
[main 2022-05-04T08:39:36.404Z] update#setState checking for updates
[main 2022-05-04T08:39:36.404Z] RequestService#request https://update.code.visualstudio.com/api/update/linux-x64/stable/dfd34e8260c270da74b5c2d86d61aee4b6d56977
[main 2022-05-04T08:39:36.404Z] resolveShellEnv(): skipped (VSCODE_CLI is set)
[main 2022-05-04T08:39:36.578Z] update#setState idle
```

**Comment by hizeph** [comment]
Attaching the `*.dmp` file

[dump.zip](https://github.com/microsoft/vscode/files/8618896/dump.zip)
s

**Comment by deepak1556** [comment]
Issue is addressed in latest stable `1.73` with Electron 19
